dashboard/src/components/TooltipDateTime/TooltipDateTime.tsx

```diff
@@ -0,0 +1,64 @@
+import { format } from 'date-fns';
+
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/Tooltip';
+import { getDateOffset } from '@/utils/utils';
+
+type TooltipDateTimeProps = {
+  dateTime: string;
+  forceNoDate?: boolean;
+  dateFormat?: string;
+  timeFormat?: string;
+  lineBreak?: boolean;
+  showTime?: boolean;
+  showTZ?: boolean;
+  showTooltip?: boolean;
+};
+
+const TooltipDateTime = ({
+  dateTime,
+  forceNoDate,
+  dateFormat,
+  timeFormat,
+  lineBreak,
+  showTime,
+  showTZ,
+  showTooltip,
+}: TooltipDateTimeProps): JSX.Element => {
+  const dateObj = new Date(dateTime);
+
+  let date = dateFormat
+    ? format(dateObj, dateFormat)
+    : dateObj.toLocaleDateString();
+  let time = timeFormat
+    ? format(dateObj, timeFormat)
+    : dateObj.toLocaleTimeString();
+  let tz = getDateOffset(dateObj);
```

Comment:
you are not redefining `tz` here so it should be const

tip: always use `const` for variable, if there is a moment that you need to redefine, change it to let when that moment arrives